docs/source/maintainers.rst

```diff
@@ -0,0 +1,63 @@
+=====================
+Maintainer Guidelines
+=====================
+
+This page describes best practices for Dask maintainers.
+
+Thank you for helping make Dask a useful library and fostering a
+welcoming community.
+
+Merging pull requests
+=====================
+
+Pull requests should be reviewed
+--------------------------------
+
+Pull requests from non-maintainers should be reviewed and approved by at
+least one maintainer before being merged.
+
+Ideally, pull requests from maintainers should also be reviewed before
+merging. However, because reviewer bandwidth is limited, maintainers will
+sometimes self-merge their own pull requests if the content of the pull
+request is viewed as uncontroversial (e.g. typo fix). If a maintainer has
+a substantial pull request which hasn't received a review, and the maintainer
+is confident in the proposed changes, they should post a final comment along
+the lines of "Merging in 24 hours if no further response" and then self-merge
```

Comment:
lazy consensus within 24hr is a bit tricky given different timezones and the fact that people are on PTO, are busy, etc. Typically one chooses rather 48h or even 72h to account for this.

Reply:
Great point -- updated this to a 48 hour minimum